ui/src/pages/frontsite/components/Document/DocumentStats.js

```diff
@@ -2,9 +2,11 @@ import React, { Component } from 'react';
 import { Icon, Table, Message } from 'semantic-ui-react';
 import { stats } from '@api/stats';
 import { recordToPidType } from '@api/utils';
-import _get from 'lodash/get';
+import get from 'lodash/get';
 
 export class DocumentStats extends Component {
+  _isMounted = false;
```

Comment:
nicely spotted! We might have this behaviour in other components that are not using Redux.
I dig a little bit on this and our React friends suggests to do something similar: https://reactjs.org/blog/2015/12/16/ismounted-antipattern.html

Maybe you could create a small reusable utility for this and we can reuse it around.

@zzacharo is this happening also in the components that you have refactored to remove Redux?

Reply:
fixed